Serhii Lakhman/Task 2/MyException.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <iostream>
+
+class MyExceptoin : public std::exception
+{
+
+public:
+	
+	MyExceptoin(const char *msg):exception(msg)
+	{
+
+		std::cout << "!!! CAnyType exception !!!" << std::endl;
```

Comment:
it is better to set some reasonable message instead of printing something from constructor